cmd/substreams/codegen.go

```diff
@@ -12,18 +14,30 @@ import (
 )
 
 var codegenCmd = &cobra.Command{
-	Use:   "codegen <package>",
+	Use:   "codegen [<manifest>]",
 	Short: "Generate a Rust trait and boilerplate code from your 'substreams.yaml' for nicer development",
-	RunE:  runCodeGen,
-	Args:  cobra.ExactArgs(1),
+	Long: cli.Dedent(`
+		Generate a Rust trait and boilerplate code from your 'substreams.yaml' for nicer development. 
+		The manifest is optional as it will try to find one in your pwd if nothing entered.
```

Comment:
```suggestion
		The manifest is optional as it will try to find one a file named 'substreams.yaml' in current working directory if nothing entered.
```